src/native/_app.js

```diff
@@ -74,19 +74,30 @@ module.exports = {
             const result = helpers.makeFfiResult(resultPtr);
             if (result.error_code !== 0) {
               reject(makeError(result.error_code, result.error_description));
-              return;
```

Comment:
Why are we removing this? the function will keep executing after calling the reject callback was invoked.